bloom-housing/bloom
# 2783/application due date sentence

# Pull Request Template

## Issue Overview

This PR addresses #2783


## Description

Updates terms text to render application due date only if it exists in listing settings.

## How Can This Be Tested/Reviewed?

1. Start an application for listing with a defined `application due date`
2. Make sure in the `terms` step the first paragraph includes the application due date
3. Start an application for listing without `application due date`
4. Make sure in the `terms` step, the first paragraph with the application due date doesn't exist

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
